lib/utils/tracking.test.js

```diff
@@ -18,11 +16,30 @@ describe('tracking', () => {
   let sendPending;
   let expectedState = 'success';
   let usedUrl;
+  let pendingRequests = 0;
+  let concurrentRequestsMax = 0;
+
+  const generateEvent = (type, timestamp = Date.now()) => {
+    let data;
+    switch (type) {
+      case 'user':
+        data = { data: { timestamp: Math.round(timestamp / 1000) } };
+        break;
+      case 'segment':
+        data = { properties: { general: { timestamp } } };
+        break;
+      default:
+        throw new Error('Unrecognized type');
+    }
+    return track(type, data);
+  };
   before(() => {
     ({ track, sendPending } = proxyquire('./tracking.js', {
       './isTrackingDisabled': false,
       'node-fetch': url => {
         usedUrl = url;
+        ++pendingRequests;
```

Comment:
Not a review comment but smth. out of curiosity.

Do you do a prefix increment because of performance reasons (at least in C(++) there's this urban legend that prefix is faster than postfix).

BTW: I tend to use prefix as well.

Reply:
I think it's leftover from my PHP days, where it was said that `++i` is faster than `i++` and then I got a habit of favoring `++i`,  but I know that in JSm performance wise it's totally irrelevant :)

Nonetheless I also find it more readable than `i++`